bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/SpectralDistribution.scala

```diff
@@ -52,7 +52,7 @@ object LibraryStar {
   case object B0V       extends LibraryStar("B0V (300nm-6.0um)",        "B0V")
   case object B05V      extends LibraryStar("B0.5V (100nm-27um model)", "B0.5V_calspec")
   case object B3V       extends LibraryStar("B3V (100nm-32um model)",   "B3V_calspec")
-  case object B5_7V     extends LibraryStar("B5-7V (300nm-6.0um)",      "b5-7v")
+  case object B5_7V     extends LibraryStar("B5-7V (300nm-6.0um)",      "B5-7V")
```

Comment:
These need to be ported to lucuma-core

Reply:
I think the code in `LibraryStar` that searches for an item by name needs to ignore case to make this work (or we write a custom migration that updates the names in the XML).  The import probably works but deletes data.

Reply:
My plan was to rename all the lower-case files so that they match the upper-case object names.  It looks like I missed changing the dash to an underscore.